news/_posts/2015-08-26-GSoC2015-Wrap-up-and-Outlook.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "GSoC2015: Wrap up and Outlook"
+authors: Amit
+date: "2015-08-26 19:27"
+---
+
+Hello everyone,
+
+During the past few weeks we have worked on improving the Documentation, and have focused on the parallelization of Quantum Monte Carlo Wave Function Method
+(please refer to the previous [blog](http://juliaquantum.github.io/news/2015/08/GSoC2015-Quantum-MCWF-method-and-QuDynamics/) for the implementation of Quantum MCWF in QuDynamics). The documentation has completely been revamped, there is a structure
+to each of the method/type implemented. The general outline is to include the following for every method/type :
+
+```julia
+@doc """
+Name of the method/type
+
+Requirement for the method/type (or) the action of method/type
+
+### Fields for type / ### Arguments for method
+
+* Field/Argument name :: Type
+
+Description of Field/Argument
+""" ->
+```
+
+The idea has been taken from the [Sims.jl](https://github.com/tshort/Sims.jl) and we hope to style the documentation of the
+future types/methods in this fashion. Moving on, we have a revamped [README.md](https://github.com/JuliaQuantum/QuDynamics.jl) which
+has more content on the installation of the package, usage of the package and references to
+the design of the package.
+
+We would like to present the overview of the package :
+
+The following Dynamical Equations can be solved using this package :
+
+1. Schrodinger Equation
+2. Liouville von Neumann Equation
+3. Lindblad Master Equation
+
+Using the following solvers :
+
+1. Euler Method, Crank-Nicholson Method, Krylov subspace Method
+2. ode45, ode78, ode23s which have been integrated from [ODE.jl](https://github.com/JuliaLang/ODE.jl)
+3. Two versions of `expmv` implementations from :
+   a. [ExpmV.jl](https://github.com/marcusps/ExpmV.jl)
+   b. [Expokit.jl](https://github.com/acroy/Expokit.jl)
+4. Quantum Monte-Carlo Wave Function Method has been integrated with reference from the article : Monte Carlo Simulation of the Atomic Master Equation for Spontaneous Emission, Phys. Rev. A 45, 4879 (1992) by R. Dum, P. Zoller, and H. Ritsch.
+
+Please refer to the [usage](https://github.com/amitjamadagni/QuDynamics.jl/blob/master/README.md#usage) of the README for further examples.
+
+We have worked on the parallelization of MCWF method and here is an outline on how parallelization would look :
+
+```julia
```

Comment:
I think you should mention that this part has to be put into some file for convenience.